examples/step-90/doc/intro.dox

```diff
@@ -0,0 +1,95 @@
+<i>
+This program was contributed by Vladimir Yushutin.
+
+The material is based upon work partially supported by NSF.
+</i>
+
+<a name="Intro to step-90"></a>
+<h1>Introduction</h1>
+
+In this example, we implement the trace finite element method (TraceFEM) in deal.II.
+Two aspects are of our focus. First, the surface approximation is separated from the discretization of the surface PDE,
+ e.g. a $Q_2$ discrete level-set and a $Q_1$ solution are possible, but both should be based on the same bulk triangulation.
+ Second, we make sure that performance of TraceFEM in the parallel implementation corresponds to that of a classical
+  fitted FEM for a two-dimensional problem. We demonstrate how to achieve both goals by using a combination of MeshWorker
+  and NonMatching capabilities.
+
+<h3>A non-trivial surface</h3>
+One of the possible bottlenecks in achieving the two-dimensional complexity while solving a surface PDE with TraceFEM
+is the surface approximation. The level-set function must be defined in a three-dimensional domain and we will create
+a mesh which is localized near the zero contour line, i.e near the surface, to restore the two-dimensional complexity.
+To demonstrate this we choose a non-trivial surface $\Gamma$ given by
+ @f{equation*}
+   \frac{x^2}{4}+ y^2 +
+                \frac{4  z^2}
+                  {(1 + 0.5  \sin(\pi  x))^{2}} =
+                1
+ @f}
+ We note that the surface's curvature varies significantly.
+
+<h3>Model problem</h3>
+
+We would like to solve the simplest possible problem, Laplace--Beltrami equation,
+@f{equation*}
+  -\Delta_\Gamma u + u = f \qquad  \text{in }\, \Gamma,
+@f}
+ Here we added the term $u$ to the left-hand side so the problem becomes well-posed.
```

Comment:
It's ok to leave it as is, but do want to point out that that's a mathematician's approach :-) From an applied perspective, the goal is to solve equations that are relevant because they describe *real* situations in physics, chemistry, or some other discipline. The equation is what it is, i.e., what came out of a modeling process. Whether or not we *like* the equation is not a relevant question, and similarly we cannot just *choose* to add a term to an equation to make the problem easier to solve.

That said, this is a *tutorial* in which we teach techniques, not solve real applications, and so I'm ok with this statement :-)

Reply:
Yes, I think this is great. Thanks for indulging me :-)